packages/integration-tests/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "sentry-browser-integration-tests",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "engines": {
+    "node": ">=10"
+  },
+  "private": true,
+  "scripts": {
+    "clean": "rimraf -g suites/**/dist",
+    "install-browsers": "playwright install --with-deps",
+    "lint": "run-s lint:prettier lint:eslint",
+    "lint:eslint": "eslint . --cache --cache-location '../../eslintcache/' --format stylish",
+    "lint:prettier": "prettier --check \"{suites,utils}/**/*.ts\"",
+    "test:ci": "playwright test ./suites --browser='all' --reporter='line'",
+    "pretest": "yarn clean",
+    "test": "playwright test ./suites"
+  },
+  "dependencies": {
+    "@playwright/test": "^1.17.0",
+    "@sentry/browser": "file:../browser",
+    "babel-loader": "^8.2.2",
+    "handlebars-loader": "^1.7.1",
+    "html-webpack-plugin": "^5.5.0",
+    "playwright": "^1.17.1",
+    "ts-loader": "^9.2.6",
+    "typescript": "^4.5.2",
```

Comment:
We need to stick with `3.7.5` to match with the rest of the codebase. Can we remove this?
```suggestion
```

Reply:
@AbhiPrasad, `@playwright/test` uses `export type` syntax for its types (and we're importing them). It's not a problem while running tests, but addressing https://github.com/getsentry/sentry-javascript/pull/3989#discussion_r767519515, to run a type check without compiling before tests, we need a newer version of TS. We also no longer need `ts-loader` as `init` and `subject` scripts are not TS. So the only use of TS here will be static type checking. So is that fine to keep this?

Reply:
For ref: https://github.com/getsentry/sentry-javascript/pull/3989/commits/c3d35b641304a097def765682b68e7aef9a82d20

Reply:
Talked with @AbhiPrasad, and we agreed we can address this later in a follow up.